app/controllers/bookings_controller.rb

```diff
@@ -1,5 +1,5 @@
 class BookingsController < ApplicationController
-  before_action :find_dress
+  before_action :find_dress, only: %i[new create destroy]
```

Comment:
destroy is in so that the flash has the instance of dress in the message.

Reply:
Ok that sounds good then